static/js/message_list.js

```diff
@@ -427,6 +427,10 @@ exports.MessageList.prototype = {
         return this.data.get_last_message_sent_by_me();
     },
 
+    get_first_message_of_stream_topic: function (stream, topic) {
```

Comment:
We generally try to use e.g. `stream_name` or `stream_id` for streams in local variables for clarity.